controllers/app.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("../models/app.models")
+
+exports.getTopics = (req, res, next) => {
+    selectTopics().then((rows) => {
+        return res.status(200).send(rows)
```

Comment:
We should be sending our response in an object on a suitable key 🔑

Reply:
As above, put it on to a key of topics. Changed tests to accommodate and retested them.